apps/api/src/app/execution-details/usecases/create-execution-details/create-execution-details.command.ts

```diff
@@ -4,6 +4,7 @@ import { EnvironmentWithSubscriber } from '../../../shared/commands/project.comm
 import { JobEntity } from '@novu/dal';
 
 export enum DetailEnum {
+  CHAT_WEBHOOK_URL_MISSING = 'Webhook URL for the chat channel is missing',
```

Comment:
Is the message clear enough? Should we mention `subscriber` as it could relate to it?